### ISSUE TITLE
Smooth effect doesn't work in Chrome/Edge. Jumps straight to section.

### ISSUE THREAD

**Issue by s7aks** [freddydumont/gatsby-plugin-smoothscroll#9]
Newb here, pretty new to Gatsby/React, somewhat familiar with development.
For me it works only in Firefox. In Chrome/Edge it jumps straight to the section, without the smooth effect.

I have a "ScrollBtn" component...
```
import React from 'react';
import styled from 'styled-components';
import scrollTo from 'gatsby-plugin-smoothscroll';

const ScrollBtn = () => {
    return(
        <Button onClick={() => scrollTo('#main')}>
            <BtnText>SCROLL DOWN</BtnText>
            <Icon viewBox="0 0 12.148 18.918">
                <path d="dfgdgdfgdfg" transform="translate(-16.275 -32)"/>
            </Icon>
        </Button>
    );
}
```

imported into the hero component
```
return(
        <HeroWrapper>
            <HeroImage ref={animationContainer} />
            <HeroArea>
                <TextA>Lorem ipsum, yada yada</TextA>
                <TextB>More Lorem ipsum, yada yada</TextB>
                <ContactBtn />
                <CoolBtn />
                <LitBtn />
            </HeroArea>
            <ScrollBtn />
        </HeroWrapper>
    );
```

, which scrolls to the main section.
```
const Main = () => {
    return(
        <MainWrapper id="main">
            <ImageA />
            <ProjectsWrapper>
                <ProjectA />
                <ProjectB />
                <ProjectC />
            </ProjectsWrapper>
            <ImageB />
        </MainWrapper>
    );
}
```

I'm using 12 column grid and wrappers for each section.

**grid.js**
```
export const Wrapper = styled.div`
  height: 100%;
  display: grid;
  grid-template-columns: repeat(12, 1fr);
  grid-template-rows: 70px 830px repeat(19, 100px);
  column-gap: 16px;
  margin: 50px 131px 0px 131px;
`;
```
**heroWrapper.js**
```
export const HeroWrapper = styled.div`
    grid-column: 1 / 13;
    grid-row: 2 / 3;

    -webkit-animation: fadein 1.5s; /* Safari, Chrome and Opera */
       -moz-animation: fadein 1.5s; /* Firefox */
        -ms-animation: fadein 1.5s; /* Internet Explorer */
         -o-animation: fadein 1.5s; /* Opera */
            animation: fadein 1.5s;
`;
```

**mainWrapper.js**
```
export const MainWrapper = styled.div`
    padding-top: 50px;
    grid-column: 1 / 13;
    grid-row: 3 / 20;
`;
```

I've been googling all day with no luck,

I tried [Anchor links](https://www.gatsbyjs.com/plugins/gatsby-plugin-anchor-links/) but it does the same thing.
I also tried [this gatsby starter](https://www.gatsbyjs.com/starters/desmukh/gatsby-starter-woo/) which uses the same plugin for the navbar , but again the smooth effect works only in Firefox. The strange thing is that it uses Anchor Links as well(Learn More btn), which works in the demo and also local when I pull it. When I apply the same syntax on the thing I'm working it doesn't work.

Confused, but also kinda sure the problem is on my side.

**Comment by freddydumont** [comment]
Did you add the plugin to your `gatsby-config.js`?
```javascript
plugins: [`gatsby-plugin-smoothscroll`];
```

**Comment by s7aks** [comment]
Yup, it's added.

**Comment by s7aks** [comment]
I believe I got it fixed.

I had to set smooth scrolling in ://flags from "Default" to "Enabled" for each browser.
Why was it set to "Default" in Chrome/Edge and Opera(fresh install) remains a mystery.

So, for anyone who has the same problem:

Go to:
chrome://flags
opera://flags
edge://flags

Search for:
"Smooth Scrolling"

Set it:
from "Default" to "Enabled"

In Firefox smooth scrolling is enabled by default straight from a fresh install, but If someone encounters this problem in Firefox.
Settings -> Search for Smooth Scrolling(Browsing section) -> "Auto scrolling" and "Smooth scrolling" and check those suckers.

Closing, god damnit.........
